pkg/providers/httpjson/httpjson.go

```diff
@@ -0,0 +1,138 @@
+package httpjson
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	// nolint
```

Comment:
nolint?

Reply:
Removed in favor of:

gci write -s standard -s default -s "prefix(github.com/helmfile/vals)" pkg/providers/httpjson/httpjson.go